core/src/repair_service.rs

```diff
@@ -279,6 +288,18 @@ impl RepairService {
                 repair_weight.set_root(new_root);
                 set_root_elapsed.stop();
 
+                // Remove dumped slots from the weighting heuristic
+                dump_slots_elapsed = Measure::start("dump_slots_elapsed");
+                dumped_slots_receiver
+                    .try_iter()
+                    .for_each(|slot_hash_keys_to_dump| {
+                        for (slot, _correct_hash) in slot_hash_keys_to_dump {
+                            // TODO: actually do something with this correct hash
```

Comment:
Repair is hash agnostic, if we fix that we could immediately add the dumped slot as an orphan